mkinitrd

```diff
@@ -116,6 +117,14 @@ check_progs() {
     fi
 }
 
+check_kmods() {
+    local arr=("$@")
+    for d in "${arr[@]}"
+    do
+        grep -qw $d "/lib/modules/$KVER/modules.builtin" || grep -qw $d "/lib/modules/$KVER/modules.dep" || die "ERROR: Kernel module $d is missing"
```

Comment:
We also need to check for aliases, e.g. ext4 is an alias for ext2 but `--filesystems ext2` doesn't work because [dracut-install.c](https://github.com/dracutdevs/dracut/blob/master/install/dracut-install.c) doesn't handle aliases for builtin modules (modules.builtin.modinfo).